scaleway/resource_security_group.go

```diff
@@ -29,6 +39,25 @@ func resourceScalewaySecurityGroup() *schema.Resource {
 				Required:    true,
 				Description: "The description of the security group",
 			},
+			"stateful": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Mark security group as stateful",
+			},
+			"inbound_default_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Default inbound traffic policy for this security group",
+				ValidateFunc: supportedDefaultTrafficPolicy,
+			},
+			"outbound_default_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Default outbound traffic policy for this security group",
+				ValidateFunc: supportedDefaultTrafficPolicy,
```

Comment:
Could this just use [`validation.StringInSlice`](https://github.com/hashicorp/terraform/blob/b54cc1d95f95587e95f0d4a0a50febcfe12618ca/helper/validation/validation.go#L130)?